assignment/index.html

```diff
@@ -18,6 +18,55 @@
 
     <!--Your code starts here-->
     <script>
+
+    //MANUALLY CREATED THE ARRAY FOR HEALTH CENTERS FROM health_centers.js:
```

Comment:
The goal was to programmatically create an array of arrays.

You are close, but are still missing the header, which would be something like, `["LAT", "LNG", NAME]`